tc/core/polyhedral/memory_promotion.h

```diff
@@ -35,15 +35,25 @@ enum class AccessType : short { Read, Write };
 // ScheduleTree*), the user is responsible for maintaining the correspondance
 // between schedule tree positions and footprints.
 // Overapproximates one dimension by its lower bound, affine function of
-// parameters and schedule dimensions visible around the scope, and by a
-// constant size.
+// parameters and schedule dimensions visible around the scope, by a
+// constant size, and by a pair offset/stride for strided accesses.  If the
+// access is not strided, then "offset" is a zero expression and "stride" is 1.
+// The lowerBound and the size are computed after removing the eventual stride.
```

Comment:
The stride is computed first, so it's not "eventual".
Did you mean "potential"?